src/pages/Todo/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useState } from "react";
+import { TextField, Button, Box } from "@mui/material";
+import { HeaderProps } from "../types";
+import { useSnackbarAlerts } from "../../../hooks/useSnackbarAlerts";
+
+const Header: FC<HeaderProps> = ({ addTask }) => {
+  const { showSuccessSnackbar } = useSnackbarAlerts();
+
+  const [taskName, setTask] = useState<string>("");
+  const [deadline, setDeadline] = useState<number>(1);
+
+  const handleAdd = () => {
+    if (taskName.trim() === "") return;
+    addTask(taskName, deadline);
+    setTask("");
+    setDeadline(0);
```

Comment:
Why we dont reset to 1 if the initial value is 1?
```suggestion
    setDeadline(1);
```